CryptoSwift.xcodeproj/project.pbxproj

```diff
@@ -1192,6 +1208,25 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
```

Comment:
maybe don't add BigInt as a dependency here, but use cocoa pods instead with the post_install hook.